dtfs-central-api/src/v1/controllers/utilisation-report-service/get-selected-fee-records-details.controller/helpers.ts

```diff
@@ -26,12 +40,16 @@ export const mapToSelectedFeeRecordDetails = async (
     throw new NotFoundError(`Failed to find a bank with id '${bankId}'`);
   }
 
+  // On the premium payments tab it is only possible to select fee records with no linked payments or a single group of fee records all having the same linked payments
+  const distinctPaymentsForFeeRecords = selectedFeeRecordEntities[0].payments;
+  const recordedPaymentDetails = distinctPaymentsForFeeRecords.map((paymentEntity) => mapPaymentEntityToSelectedFeeRecordsPaymentDetails(paymentEntity));
   const selectedFeeRecordDetails = selectedFeeRecordEntities.map((feeRecordEntity) => mapFeeRecordEntityToSelectedFeeRecordDetails(feeRecordEntity));
 
   return {
     bank: { name: bankName },
     reportPeriod,
     totalReportedPayments: getTotalReportedPayments(selectedFeeRecordDetails),
     feeRecords: selectedFeeRecordDetails,
+    payments: recordedPaymentDetails ?? [],
```

Comment:
I _think_ `recordedPaymentDetails` will always be an array - if you enable the `payments` relation on the query and there are no related payments, then `payments` will be an empty array, so I think `recordedPaymentDetails` will always be an array, so the nullish coalesce operator isn't necessary

Reply:
Ah yes nice, I think I added it at one point to try and get rid of a bug but clearly wasn't needed or things changed 😄